README.md

```diff
@@ -1,35 +1,33 @@
 # Docs @ Docker
 
 <div align="center">
-<img src="static/assets/images/docker-docs.svg" alt="Welcome to Docker Documentation">
+<img src="static/assets/images/docker-docs.svg" alt="Docker Documentation Logo"/>
 </div>
 <br/>
 <br/>
 
-Welcome to the Docker Documentation repository. This is the source for the [Docker Docs Website](https://docs.docker.com/).
+Welcome to the Docker Documentation repository. This is the official Docker documentation.
 
-Feel free to open pull requests or issues. Our docs are completely open source, and we deeply appreciate contributions from the Docker community!
+Feel free to open pull requests or issues. Our docs are completely open source and encourage community contributions.
 
-## Provide feedback
+## How to Provide Feedback
 
 We’d love to hear your feedback! To submit feedback:
-- Click **[New issue](https://github.com/docker/docs/issues/new)** on the docs repository, or
-- Click **Request changes** in the right column of every page on
-  [docs.docker.com](https://docs.docker.com/), or
+- Click **[New issue](https://github.com/docker/docs/issues/new)** on the docs GitHub repository.
+- Click **Request changes** in the right column of every page on [docs.docker.com](https://docs.docker.com/).
 - Click **Give feedback** on every page in the docs.
 
-To get community support, use the [Docker Community Slack](https://dockr.ly/comm-slack). Personalized support is available
-through the Docker Pro, Team, and Business subscriptions. See [Docker
-Pricing](https://www.docker.com/pricing) for details.
+To get community support, use the [Docker Community Slack](https://dockr.ly/community-support) through the Docker Pro, Team, and Business subscriptions. See [Docker Pricing](https://www.docker.com/pricing) for details.
```

Comment:
Please revert this change as it's changed the fundamental meaning of the sentence